src/app/utils/tests/Comparable.spec.ts

```diff
@@ -43,9 +43,10 @@ describe('Comparable', () => {
             expect(comparableEquals(object1, object3)).toBeFalse();
         });
         it('should fail if objects are not comparable', () => {
-            spyOn(Utils, 'handleError');
-            comparableEquals(new DummyNonComparableObject(5), new DummyNonComparableObject(5));
-            expect(Utils.handleError).toHaveBeenCalledWith('Comparing non comparable objects');
+            const expectedError: string = 'Comparing non comparable objects: DummyNonComparableObject and DummyNonComparableObject';
```

Comment:
GWT Si.you.plé

Reply:
aussi, le \n ligne 51 est pas nécessaire

Reply:
done both